src/utils/SVreq.js

```diff
@@ -3,9 +3,30 @@ const SV = new google.maps.StreetViewService();
 export default function SVreq(loc, settings) {
     return new Promise(async (resolve, reject) => {
         if (!loc.panoId) {
-            await SV.getPanoramaByLocation(new google.maps.LatLng(loc.lat, loc.lng), settings.radius, checkPano).catch((e) =>
-                reject({ loc, reason: e.message })
-            );
+            if(settings.changeToOfficial && settings.rejectUnofficial) {
+                let returnLoc = await SV.getPanorama({
+                    location: {lat: loc.lat, lng: loc.lng},
+                    preference: google.maps.StreetViewPreference.NEAREST, // Set the preference
+                    source: google.maps.StreetViewSource.OUTDOOR, // Get outdoor panoramas
```

Comment:
Earlier this year Google added a `StreetViewSource.GOOGLE` flag. The main difference is that it allows official "non-outdoor" coverage, which (*IIRC*) makes a difference for Mongolia gen 3 and some Korea car coverage erroneously marked as indoor trekker
```suggestion
                    sources: [google.maps.StreetViewSource.GOOGLE], // Only search official panoramas
```